test/keypoints/CMakeLists.txt

```diff
@@ -1,9 +1,23 @@
-PCL_ADD_TEST(keypoints_general test_keypoints
-                 FILES test_keypoints.cpp
-                 LINK_WITH pcl_gtest pcl_io pcl_kdtree pcl_filters pcl_keypoints
-                 ARGUMENTS "${PCL_SOURCE_DIR}/test/cturtle.pcd")
+set(SUBSYS_NAME tests_keypoints)
+set(SUBSYS_DESC "Point cloud library keypoints module unit tests")
+set(SUBSYS_DEPS global_tests keypoints common search kdtree octree features filters)
```

Comment:
Sorry for the massive delay :sweat_smile: . I found a way to mimic the behaviour you described, to I'm applying the changes now. The doxygen issue after...